CharacterCreation/06_01_Systemized_Skills.txt

```diff
@@ -1,68 +1,125 @@
 ==============================
-Full list of skills
+Skills
 ==============================
+	Each character learns things while going through life. Maybe your time in
+the military taught you how to navigate using the stars or maybe your time working
+at a mechanic's shop made you good at repairing things? Or maybe you're one of
+those people who is just naturally good at hacking computers.
 
-* Acrobatics
-* Athletics
-* Deception
-* Insight
-* Intimidation
-* Investigation
-* Medical
-* Navigation
-* Space
-* Geography
-* Persuasion
-* Repair
-* Sleight of Hand 
-* Spotting
-* Stealth
-* Survival
+	Whenever you roll to see if your character succeeds in a task they have 
+skill in, add both your character's points in relevant skills and the relevant
+stat modifier. Ask your GM to confirm what skills and stats apply to the roll.
+
+===== Gaining Skill Points and Skill Proficiency =====
+	Each time a character levels up, they gain a certain number of skill
+points to distribute as they choose. However, a character may only gain up to 5
+points in a skill per level, up to a maximum of 60. Some feats, backgrounds and
+class passives will give a character points in a specific skill. While these
+skill gains are not subjected to the limit on skill point gain per level, they
+do contribute to the maximum of 60 points in a skill.

```

Comment:
Very good explanation! Clear, concise, all in one place! This is going to make onboarding new players _way_ easier! I didn't know how bad we needed this until I saw it.